packages/block-library/src/template-part/edit/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * WordPress dependencies
+ */
```

Comment:
When I select the template part using the block navigator the height of the block does not include its content:
![image](https://user-images.githubusercontent.com/11271197/71296843-f4e64200-2378-11ea-92d5-90a43eb25fae.png)

Reply:
I can't replicate this. Is there anything specific you are doing that causes it?

I also don't think it's related to this PR since we are just returning the regular `InnerBlocks` markup.